src/sassPartials/_shared_style.scss

```diff
@@ -13,7 +13,7 @@
     $cta: !important;
   }
 
-  @include transition;
+  @include auro_transition;
```

Comment:
This is being used by ods-button? If so, this should stay with the deprecated one.